rocksdb_replicator/tests/rocksdb_replicator_test.cpp

```diff
@@ -348,6 +361,126 @@ TEST(RocksDBReplicatorTest, 1_master_2_slaves_chain) {
   EXPECT_EQ(db_slave_2->GetLatestSequenceNumber(), 2 * n_keys);
 }
 
+TEST(RocksDBReplicatorTest, 1_master_1_slave_upstream_itself) {
```

Comment:
I think, for the test name, we still keep the camel case, not `1_master_1_slave_upstream_itself`

Reply:
I am using this style for consistency with all existing tests in this file. If we decide to use CamelCase, then we can change it for all